libc/syscalls.c

```diff
@@ -29,22 +30,25 @@ static const uint64_t _SEC_TO_MSEC = 1000UL;
 static const uint64_t _MSEC_TO_USEC = 1000UL;
 static const uint64_t _MSEC_TO_NSEC = 1000000UL;
 
-static long _syscall_mmap(long n, ...)
+OE_DEFINE_SYSCALL6(SYS_mmap)
 {
     /* Always fail */
-    OE_UNUSED(n);
     return EPERM;
 }
 
-static long _syscall_clock_gettime(long n, long x1, long x2)
+OE_DEFINE_SYSCALL2(SYS_munmap)
 {
-    clockid_t clk_id = (clockid_t)x1;
-    struct timespec* tp = (struct timespec*)x2;
+    /* Always fail */
+    return EPERM;
+}
+
+OE_DEFINE_SYSCALL2(SYS_clock_gettime)
+{
+    clockid_t clk_id = (clockid_t)arg1;
```

Comment:
```suggestion
    clockid_t clock_id = (clockid_t)arg1;
```
Avoid abbreviations, especially since the type spells it out